src/app/pages/preferences/page/forms/customtheme/customtheme.component.ts

```diff
@@ -96,72 +96,78 @@ export class CustomThemeComponent implements OnInit, OnChanges, OnDestroy {
       class:'general',
       width:'300px',
       config:[
-        { 
-          type: 'input', 
-          name: 'name', 
+        {
+          type: 'input',
+          name: 'name',
           width:'100%',
           placeholder: 'Custom Theme Name',
           required:true,
-          tooltip: 'Enter a name to identify your new theme.',
+          tooltip: 'Enter a name to identify the new theme.',
         },
-        { 
-          type: 'input', 
-          name: 'label', 
+        {
+          type: 'input',
+          name: 'label',
           width:'100%',
           placeholder: 'Menu Label',
           required:true,
-          tooltip: 'Specify how the theme name should appear in the menu.',
+          tooltip: 'Enter how the theme name will appear in the menu.',
         },
-        { 
-          type: 'select', 
-          name: 'labelSwatch', 
+        {
+          type: 'select',
+          name: 'labelSwatch',
           width:'100%',
-          placeholder: 'Menu Swatch', 
+          placeholder: 'Menu Swatch',
           required:true,
           options:this.colorOptions,
-          tooltip: "Choose which color from the palette will be used for the label swatch that appears left of the label in the menu.",
+          tooltip: "Choose which color from the palette will be used for \
```

Comment:
This is confusing, probably due to the use of directions.  Maybe
"Choose the color to display next to the label in the menu."